src/cl/prysm/prysm_launcher.star

```diff
@@ -245,8 +249,11 @@ def get_beacon_config(
 		# ^^^^^^^^^^^^^^^^^^^ METRICS CONFIG ^^^^^^^^^^^^^^^^^^^^^
 	]
 
-	if bootnode_context != None:
-		cmd.append("--bootstrap-node="+bootnode_context.enr)
+	if bootnode_contexts != None:
+		cmd.append("--bootstrap-node="+",".join([ctx.enr for ctx in bootnode_contexts]))
```

Comment:
@barnabasbusa  is comma separation allowed here?

Reply:
No, it expects:
```
"--bootstrap-node=enr:-Iq4QGg2N5AtT1hgyGdVccfAYbq9znCNEfTjv2UB_XRGhGX7erSDt0GtmAYjTvdmeguInIbRz33UvYa229fcU5u8XcyGAYm6lc7RgmlkgnY0gmlwhJ9B7teJc2VjcDI1NmsxoQPZLBX2rEiQ20AfN2f1SQLfnGMQd_t7Rq0-HNkYA9d4uoN1ZHCCIyk",
 "--bootstrap-node=enr:-KG4QGiIjyJ5YR7oq1ywJCN1SqGeq4A-2KM_MLMUmpA8cYnDDwm-Yh5m-ho0VRgCAkifqGBmCQj-USg6-J2o_IL1vvsDhGV0aDKQ7sFe4kB1lzIFAAAAAAAAAIJpZIJ2NIJpcISfQe7XiXNlY3AyNTZrMaED674M_J35Yi28giDrthajUkweV7ZKMaM23Oi1Ly3Qd0CDdGNwgiMog3VkcIIjKA",
 "--bootstrap-node=enr:-Ly4QPxn8C8ntYqXK0xSixfPxMREShVOID7fB8cR6o-CSmjaIIhaRWudih6RWYQQOgXd0DV9lnWTt4B6xr2cnH-LQs0Bh2F0dG5ldHOIAAAAAAAAAACEZXRoMpDuwV7iQHWXMgUAAAAAAAAAgmlkgnY0gmlwhES3bfCJc2VjcDI1NmsxoQPJZ3dD0de6oFg9huvvm-9tB6xL3CVws8Twj9f3lXHVqYhzeW5jbmV0cwCDdGNwgiMog3VkcIIjKA",
```